### DISCUSSION TITLE
thanhle.blog/hanh-trinh-di-tim-ly-do

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#83] [Ideas]
# thanhle.blog/hanh-trinh-di-tim-ly-do

…cho đến khi mình không được làm lead nữa. Cũng chả có lý do gì cụ thể, cũng chả ai nói gì mình cả, chỉ là những dự án sau, mình cảm nhận mình là người khá phù hợp nhưng người được chọn làm lead không còn là mình nữa.

https://thanhle.blog/blog/hanh-trinh-di-tim-ly-do

**Comment by hai123vn** [comment]
quá hay anh ơi, có 1 vài chỗ bị sai chính tả nhưng mà vẫn đọc đc 🤣

**Reply by thanhlmm** [reply]
lâu rồi ko viết nhiều nên tay run e :)))

**Reply by hai123vn** [reply]
em có ham quá khi anh có thể 1 tuần ra 1 bài không anh. Về nội dụng đầu tư hoặc tiết kiệm như chứng khoán, chứng chỉ quỹ hoặc cách tiết kiệm nào đó hợp lý ạ

**Reply by thanhlmm** [reply]
Mấy cái này thì em tham khảo thử kênh youtube của anh Hiếu tv, hình như anh Hiếu cũng bán khóa học :))))

Anh thì đi uống thì nói dc thôi chứ mấy cái này không viết tốt :))))

**Comment by khiem20tc** [comment]
``` làm việc lúc ôm cũng chả có gì là hiệu quả ``` 
em cũng muốn được làm việc lúc ôm :")

**Reply by thanhlmm** [reply]
Chà a sai quá, làm việc lúc ôm hẳn là phải hiệu quả hơn rồi

**Comment by htoann** [comment]
Hi anh ạ, em là người đã có tâm sự với anh về lần OT 3 tháng ở công ty thì em đã quyết định nghỉ cũng gần được 1 tháng rồi. Thực sự mà nói thì sau khoảng 1 2 tuần em bắt đầu thấy dường như em đã quyết định sai. Em xin nghỉ vì lý do chỉ là muốn trốn tránh khỏi áp lực, khỏi những bug features của em chưa kịp fix, khỏi những lời phàn nàn của đồng nghiệp, sếp, khỏi những lời khuyên nhủ mà người ta chỉ muốn tốt cho mình. Em đang dự định sẽ xin sếp của em cho em 1 cơ hội để có thể trở về với team. Thật sự tốt nếu có 1 lời khuyên nho nhỏ nào đó từ anh.

**Reply by thanhlmm** [reply]
Hi em,

Anh nghĩ em nên nghỉ thêm và suy nghĩ kĩ thử xem. Maybe là em cần thời gian làm công việc khác để trau dồi thêm, maybe là em cần thêm thời gian để bớt stress lại.
Vì sẽ vẫn có trường hợp em quay lại làm và mọi việc vẫn rối như vậy mà. Ý định tốt nhưng năng lực không đủ thì cũng gây thiệt hại lắm (cho cả cty và bản thân em).
Bản thân cũng không cần thắng mọi trận chiến mà thằng cả cuộc chiến mới quan trọng
